static/app/views/performance/browser/webVitals/pageOverviewWebVitalsDetailPanel.tsx

```diff
@@ -17,31 +17,30 @@ import {getDuration} from 'sentry/utils/formatters';
 import {PageAlert, PageAlertProvider} from 'sentry/utils/performance/contexts/pageAlert';
 import {getTransactionDetailsUrl} from 'sentry/utils/performance/urls';
 import {generateProfileFlamechartRoute} from 'sentry/utils/profiling/routes';
+import useReplayExists from 'sentry/utils/replayCount/useReplayExists';
 import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 import {useRoutes} from 'sentry/utils/useRoutes';
 import {PerformanceBadge} from 'sentry/views/performance/browser/webVitals/components/performanceBadge';
 import {WebVitalDetailHeader} from 'sentry/views/performance/browser/webVitals/components/webVitalDescription';
 import {WebVitalStatusLineChart} from 'sentry/views/performance/browser/webVitals/components/webVitalStatusLineChart';
-import {
-  calculatePerformanceScoreFromTableDataRow,
-  PERFORMANCE_SCORE_MEDIANS,
-  PERFORMANCE_SCORE_P90S,
-} from 'sentry/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/calculatePerformanceScore';
+import {calculatePerformanceScoreFromTableDataRow} from 'sentry/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/calculatePerformanceScore';
 import {useProjectRawWebVitalsQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsQuery';
 import {useProjectRawWebVitalsValuesTimeseriesQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/rawWebVitalsQueries/useProjectRawWebVitalsValuesTimeseriesQuery';
 import {calculatePerformanceScoreFromStoredTableDataRow} from 'sentry/views/performance/browser/webVitals/utils/queries/storedScoreQueries/calculatePerformanceScoreFromStored';
 import {useProjectWebVitalsScoresQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/storedScoreQueries/useProjectWebVitalsScoresQuery';
-import {useTransactionSamplesWebVitalsQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/useTransactionSamplesWebVitalsQuery';
+import {useGoodMehAndBadInteractionsSamplesQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/useGoodMehAndBadInteractionsSamplesQuery';
+import {useGoodMehAndBadTransactionsSamplesQuery} from 'sentry/views/performance/browser/webVitals/utils/queries/useGoodMehAndBadTransactionsSamplesQuery';
```

Comment:
Oof these two names are a little confusing. I had to really focus on what the difference was and realize they weren't duplicates

Could these be better names? I think it might be clearer if we have interactions vs transactions highlighted up front. And I don't know if we need to be so explicit about using `GoodMehAndBad` in the naming. I think we can add that as a comment to the hook explaining that those are the categories and autocomplete can tell us what categories they are.
`useInteractionsCategorizedSamplesQuery`
`useTransactionsCategorizedSamplesQuery`

or `useCategorizedXSamplesQuery` could make it clearer too.

Reply:
You can follow this up in another PR

Reply:
I went ahead and cleaned up the names!